bin/configure-tower-projects.py

```diff
@@ -52,6 +52,15 @@
     "m6i.8xlarge", "r5a.8xlarge", "r6a.8xlarge", "r6i.8xlarge"
 )
 
+GPU_EC2_INSTANCE_TYPES = (
+    # Amazon linux 2023 AMIs:
+    #   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html
+    # GPU instance types:
+    #   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-gpu.html
+    "p4d.24xlarge", "p4de.24xlarge", "p5.48xlarge", "p5e.48xlarge",
+    "p5en.48xlarge", "p6-b200.48xlarge", "p6e-gb200.36xlarge"
```

Comment:
Thanks for adding these! Can we also include the cheaper GPU alternatives too? 😄 

```
"g5.xlarge", "g5.2xlarge", "g5.4xlarge", "g5.8xlarge", "g5.16xlarge"
```

Reply:
are you sure @vpchung ?  I thought `P` family is what you guys wanted.  Here's an good article about the difference between `P` and `G` families of GPUs. https://www.cloudoptimo.com/blog/aws-p-family-vs-g-family-a-detailed-comparison-for-high-performance-tasks/

Reply:
After thinking on this more, I think you are right, @zaro0508 , let's stick with what you added.